src/test/java/core/basesyntax/ReportTests.java

```diff
@@ -0,0 +1,59 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Operations;
+import core.basesyntax.service.impl.FruitStorageServiceImpl;
+import core.basesyntax.strategy.BalanceHandler;
+import core.basesyntax.strategy.OperationHandler;
+import core.basesyntax.strategy.OperationStrategy;
+import core.basesyntax.strategy.OperationStrategyImpl;
+import core.basesyntax.strategy.PurchaseHandler;
+import core.basesyntax.strategy.ReturnHandler;
+import core.basesyntax.strategy.SupplyHandler;
+import java.util.HashMap;
+import java.util.Map;
+import junit.framework.TestCase;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class ReportTests extends TestCase {
+    private static FruitStorageServiceImpl reportCreator;
+
+    @Override
+    public void setUp() throws Exception {
+        Map<Operations, OperationHandler> operationMap = new HashMap<>();
+        operationMap.put(Operations.B, new BalanceHandler());
+        operationMap.put(Operations.P, new PurchaseHandler());
+        operationMap.put(Operations.R, new ReturnHandler());
+        operationMap.put(Operations.S, new SupplyHandler());
+        OperationStrategy operationStrategy = new OperationStrategyImpl(operationMap);
+        reportCreator = new FruitStorageServiceImpl(operationStrategy);
+        super.setUp();
+    }
+
+    @Test
+    public void test1Report_Ok() {
+        reportCreator.saveFruitToStorage("src/test/resources/test1_correct.csv");
+        String actual = reportCreator.createReport();
```

Comment:
test these methods independently of each other - remember that you can access Storage any time

and you should clear the Storage after each test